docs/governance/governance.md

```diff
@@ -0,0 +1,163 @@
+<!-- 
+A project charter discusses _what it is and why it exists_, a governance discusses _how it operates_.
+-->
+
+# Project Governance
+
+## 1. Guiding Principles
+<!-- Optional section to state high-level principles like openness, meritocracy, consensus, etc. -->
+The value of EESSI grows exponentially with two things: the amount of systems that make the EESSI software stack available, and the amount of software that is available in the EESSI software stack. Thus, the first goal of this governance is to make sure everyone in the community feels sufficiently included so that they are willing to contribute to EESSI (rather than build their own solution).
+
+The second goal of this governance is to make clear how and by whom decisions in EESSI are taken. This is because trust in this process is important, both to infrastructure providers making the EESSI software stack is available on their systems, as well as by end-users of the software. Note that this concerns both large decisions, such as which architectures are supported in EESSI, as well as small decisions, such as whether to accept a certain contribution to add software to EESSI.
```

Comment:
"EESSI software stack" -> EESSI

"such as whether to accept a certain contribution to add software to EESSI" -> "such as making a specific software package in EESSI"

Reply:
Changed